client/src/components/Pagination.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import $ from 'jquery';
+
+const Pagination = ({ numReviews, paginate }) => {
+  const pages = [];
+  const numPages = Math.ceil(numReviews / 7);
+  for (let i = 1; i <= numPages; i += 1) {
+    pages.push(i);
+  }
+
+  const onClickHandler = (e) => {
+    e.preventDefault();
+    const currentPage = e.target.value;
+    $(`.page-button-${currentPage}`).css({
+      color: '#00A699',
+      border: 'none',
+      'font-weight': 'bold',
+    });
+    for (let i = 0; i < pages.length; i += 1) {
+      if (pages[i] === Number(currentPage)) {
+        continue;
+      } else {
+        $(`.page-button-${pages[i]}`).css({
+          color: 'grey',
+          border: 'none',
+          'font-weight': 'normal',
```

Comment:
I'm not 100% clear on what's happening here. Are you rendering all the page numbers of the pages other than the one the user is currently on?

Reply:
Sort of—I'm styling all the pages except the one the user is currently on with a particular style. Definitely not the most efficient way though; I'm hoping to refactor it if time allows.